packages/hash/frontend/src/pages/[account-slug]/entities/shared/simple-entity-editor.tsx

```diff
@@ -146,52 +140,30 @@ const splitSchema = (
 
 export const SimpleEntityEditor: VoidFunctionComponent<
   SimpleEntityEditorProps
-> = ({
-  accountId,
-  aggregateEntities: aggregate,
-  disabled,
-  readonly,
-  refetchEntity,
-  schema,
-  ...entityProps
-}) => {
-  const {
-    createLinks = undefined,
-    deleteLinks = undefined,
-    entityId = undefined,
-    entityProperties: existingProperties = undefined,
-    linkedEntities = [],
-    linkGroups: existingLinkGroups = [],
-  } = "entityProperties" in entityProps ? entityProps : {};
-
+> = ({ aggregateEntities, refetchEntity, schema, ...variableProps }) => {
   const onSubmit = (args: ISubmitEvent<any>, event: FormEvent) => {
     event.preventDefault();
-    if ("entityId" in entityProps) {
-      entityProps
-        .updateEntities([
-          {
-            accountId,
-            data: {
-              ...existingProperties,
-              ...args.formData,
-            },
-            entityId: entityProps.entityId,
+    if ("entityProperties" in variableProps) {
```

Comment:
Just for my understanding, why would the check for `entityId` no longer work here?

Reply:
It would, either is fine – I probably just rewrote it without thinking. I've updated in https://github.com/hashintel/hash/pull/752/commits/60e0ea0fb5ccc616e1a5a588bf8cf4e404afb846 to consistently use `entityProperties` as the discriminating field to avoid any confusion as to why `entityId` in one place and `entityProperties` in another, but either works.

Reply:
Understood, thanks for taking the time think I was overthinking the diff a bit here.